From 9e32ba61fd588353d473858a27bdf0cb0c508019 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Tue, 18 Jul 2017 19:48:08 +0200
Subject: main: Define global properties with 'var' instead of 'let'

Symbols that are defined with 'let' are no longer visible outside
the module that defines them. To unbreak the code base, define all
non-private properties as global.

https://bugzilla.gnome.org/show_bug.cgi?id=785084
---
 js/ui/main.js | 60 +++++++++++++++++++++++++++++------------------------------
 1 file changed, 30 insertions(+), 30 deletions(-)

diff --git a/js/ui/main.js b/js/ui/main.js
index b19c3b2..5af25ec 100644
--- a/js/ui/main.js
+++ b/js/ui/main.js
@@ -50,36 +50,36 @@ const A11Y_SCHEMA = 'org.gnome.desktop.a11y.keyboard';
 const STICKY_KEYS_ENABLE = 'stickykeys-enable';
 const GNOMESHELL_STARTED_MESSAGE_ID = 'f3ea493c22934e26811cd62abe8e203a';
 
-let componentManager = null;
-let panel = null;
-let overview = null;
-let runDialog = null;
-let lookingGlass = null;
-let wm = null;
-let legacyTray = null;
-let messageTray = null;
-let screenShield = null;
-let notificationDaemon = null;
-let windowAttentionHandler = null;
-let ctrlAltTabManager = null;
-let padOsdService = null;
-let osdWindowManager = null;
-let osdMonitorLabeler = null;
-let sessionMode = null;
-let shellAccessDialogDBusService = null;
-let shellAudioSelectionDBusService = null;
-let shellDBusService = null;
-let shellMountOpDBusService = null;
-let screenSaverDBus = null;
-let screencastService = null;
-let modalCount = 0;
-let actionMode = Shell.ActionMode.NONE;
-let modalActorFocusStack = [];
-let uiGroup = null;
-let magnifier = null;
-let xdndHandler = null;
-let keyboard = null;
-let layoutManager = null;
+var componentManager = null;
+var panel = null;
+var overview = null;
+var runDialog = null;
+var lookingGlass = null;
+var wm = null;
+var legacyTray = null;
+var messageTray = null;
+var screenShield = null;
+var notificationDaemon = null;
+var windowAttentionHandler = null;
+var ctrlAltTabManager = null;
+var padOsdService = null;
+var osdWindowManager = null;
+var osdMonitorLabeler = null;
+var sessionMode = null;
+var shellAccessDialogDBusService = null;
+var shellAudioSelectionDBusService = null;
+var shellDBusService = null;
+var shellMountOpDBusService = null;
+var screenSaverDBus = null;
+var screencastService = null;
+var modalCount = 0;
+var actionMode = Shell.ActionMode.NONE;
+var modalActorFocusStack = [];
+var uiGroup = null;
+var magnifier = null;
+var xdndHandler = null;
+var keyboard = null;
+var layoutManager = null;
 let _startDate;
 let _defaultCssStylesheet = null;
 let _cssStylesheet = null;
